packages/eas-cli/src/build/createContext.ts

```diff
@@ -44,14 +45,11 @@ export async function createBuildContextAsync<T extends Platform>({
   resourceClass: BuildResourceClass;
   message?: string;
   actor: Actor;
+  projectId: string;
```

Comment:
When you build with the `--platform=all` it's entirely possible that android and ios builds will have different projectIds based on envs, so projected here needs to read from `exp` from the build context that was resolved based on envs from this build profile.

Reply:
Good point. I need to rethink these particular commands as far as context goes.